src/answers-umd.js

```diff
@@ -160,6 +161,13 @@ class Answers {
     globalStorage.set(StorageKeys.SESSIONS_OPT_IN, parsedConfig.sessionTrackingEnabled);
     parsedConfig.noResults && globalStorage.set(StorageKeys.NO_RESULTS_CONFIG, parsedConfig.noResults);
 
+    const context = globalStorage.getState(StorageKeys.API_CONTEXT);
+    if (context && !isValidContext(context)) {
+      persistentStorage.delete(StorageKeys.API_CONTEXT, true);
+      globalStorage.delete(StorageKeys.API_CONTEXT);
+      console.error(`Context parameter "${context}" is invalid, omitting from the search.`);
```

Comment:
Sounds like you Bryan, Max, and yourself discussed validation! Was the consensus that in the failure case, we remove the invalid context from any subsequent requests and show a console error?

Reply:
That is right!

>if we validate and it's incorrect it should just not pass it to the backend (not like error out or anything)